### ISSUE TITLE
Add a "Add External Tileset" tool button to Tilesets panel

### ISSUE THREAD

**Issue by mitchcurtis** [bjorn/tiled#2882]
I had a .tsx file that I wanted to use in a map, but none of the tool buttons I was looking at were what I was after:

<img width="156" alt="Screen Shot 2020-08-09 at 10 54 29 am" src="https://user-images.githubusercontent.com/6002340/89728641-0e82e980-da2f-11ea-84ba-d7e376883a0d.png">

Then I found this:

https://discourse.mapeditor.org/t/importing-exporting-tsx/827/2

I think it would be more discoverable and intuitive to also have this option available with the other tool buttons in the Tilesets panel.

**Comment by bjorn** [comment]
I've added the action here in Tiled 1.5:

![image](https://user-images.githubusercontent.com/531764/110925274-45054780-8323-11eb-9c29-a93f0990843f.png)

I think that should close this issue?

**Comment by mitchcurtis** [comment]
Just out of curiosity, why not place it with the other toolbuttons? If it's a matter of space, do the other toolbuttons also "overflow" to that menu when the panel is resized to be too small?

I'm not sure if I would think to click that drop down button, but it could just be me.

**Comment by bjorn** [comment]
I agree that the menu isn't so obvious. I think the reason I decided to put it there was that I wasn't sure about which icon to use, though it could probably just be a "+".

Let's keep this issue open for now then and revisit this topic for 1.6. I'd like to improve asset selection in general to use the project instead of the system file dialog.

**Comment by eishiya** [comment]
> Let's keep this issue open for now then and revisit this topic for 1.6. I'd like to improve asset selection in general to use the project instead of the system file dialog.

Popping in just to comment on this. I much prefer the system file dialog, as I find it easier to navigate than the Project file list in Tiled. The Windows file dialog lets me view the files in a more compact manner, and I prefer to explicitly navigate directories over having everything in a single list with paths, helps me to orient better.

As for the icon, I think the icon for New Tileset in your screenshot would make a perfect "Add External Tileset" icon. In my Tiled, the New Tileset icon has a star instead:
![image](https://user-images.githubusercontent.com/70545646/145228964-fd358387-f4a2-4141-81c2-74815092479f.png)
I don't know if this is a version difference or a theme difference, but I think using both of these icons together for the different actions would work nicely.
